### ISSUE TITLE
Merge into symbols-tree-view

### ISSUE THREAD

**Issue by sarangjo** [nicolashainaux/symbols-tree-nav#20]
Hi there, I was recently added as a collaborator of `symbols-tree-view` and am trying to clean up the plethora of issues and PR's that are on that repository. I found your fork after looking around some of the issues and comments on there, and I think it would be great if we could try to get your awesome upgrades to the package merged into the main package! Let me know if this is something you'd be interested in.

**Comment by nicolashainaux** [comment]
Yes, that's definitely a good thing to do!
The only problem is I miss time. I'll try to check this though.

**Comment by nicolashainaux** [comment]
Also there's still a pending PR here I haven't merged yet!

**Comment by sarangjo** [comment]
Sounds good. Do you think you could open a PR from this package into the main repo once that PR is merged in? If we can complete a clean merge, then I can start going through the existing issues and resolve them if they've already been fixed here!
